atc/lidar/scanner.go

```diff
@@ -92,6 +103,9 @@ func (s *scanner) check(checkable db.Checkable, resourceTypes db.ResourceTypes,
 	}
 
 	interval := s.defaultCheckInterval
+	if resource, ok := checkable.(db.Resource); ok && resource.HasWebhook() {
```

Comment:
I'd like to avoid this type assertion. The `Checkable` interface should represent everything needed to perform a check, so if we need `HasWebhook` then can we implement this in resourceType as well and always return false.

Reply:
Added `HasWebhook` to resource type.